cli/cli.ts

```diff
@@ -2655,6 +2762,7 @@ function testSnippetsAsync(...args: string[]): Promise<void> {
         let snippets = uploader.getSnippets(source)
         // [].concat.apply([], ...) takes an array of arrays and flattens it
         let extraDeps: string[] = [].concat.apply([], snippets.filter(s => s.type == "package").map(s => s.code.split('\n')))
+        extraDeps.push("microbit")
```

Comment:
pxt.appTarget should contain the "default library"... can we get rid of this hardcoded dependency?